docs/index.md

```diff
@@ -6,37 +6,47 @@ Project documentation with&nbsp;Markdown.
 
 ## Overview
 
-MkDocs is a **fast**, **simple** and **downright gorgeous** static site generator that's geared towards building project documentation. Documentation source files are written in Markdown, and configured with a single YAML configuration file.
+MkDocs is a **fast**, **simple** and **downright gorgeous** static site
+generator that's geared towards building project documentation. Documentation
+source files are written in Markdown, and configured with a single YAML
+configuration file.
 
----
-
-**MkDocs is currently still in development.**
+!!! warning
```

Comment:
Seconded. Yup we're not at 1.0 yet, but we're using it on major projects and a wide userbase.
If stuff does change between versions we'll make sure to do so with a clear upgrade path.